src/main/kotlin/LinkedList.kt

```diff
@@ -10,18 +10,54 @@ class LinkedList<T> {
     
     fun count(): Int = size
        
-    fun insert(index: Int, obj: T) {
-        TODO("Not yet implemented")
+    operator fun get(index: Int): T {
+        var current = head
+        var currentIndex = 0
+        while (current != null && currentIndex < index) {
+            current = current.nextNode
+            currentIndex++
+        }
+        return current
     }
-    fun removeAt(index: Int) {
+    operator fun set(index: Int, obj: T) {
         TODO("Not yet implemented")
     }
-    operator fun get(index: Int): T {
-        TODO("Not yet implemented")
+    
+    fun insertFirst(value: T) {
+        head = Node(value = value, nextNode = head)
+        if (last == null) {
+            last = head
+        }
+        size++
     }
-    operator fun set(index: Int, obj: T) {
+
+    fun add(value: T) {
```

Comment:
Как я сказал, мне не нравится неконсистентность имён.
```suggestion
    fun insertLast(value: T) {
```